docs/code/MlNetCookBook.md

```diff
@@ -148,12 +148,12 @@ Label	Workclass	education	marital-status
 
 This is how you can read this data:
 ```csharp
-// Create a new environment for ML.NET operations. It can be used for exception tracking and logging, 
-// as well as the source of randomness.
-var env = new LocalEnvironment();
+// Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+// as a catalog of available operations and as the source of randomness.
+var mlContext = new MLContext();
 
 // Create the reader: define the data columns and where to find them in the text file.
-var reader = TextLoader.CreateReader(env, ctx => (
+var reader = mlContext.Data.TextReader(ctx => (
```

Comment:
>mlContext [](start = 13, length = 9)

the examples of this file are actually tests in the CookbookExamples.cs. I believe we want to keep them in sync.

Reply:
We want the cookbook to show the preferred way to load data. It would be confusing for users to see both mlContext.Data.TextReader and TextLoader.CreateReader. I'll change the test instead.

---
In reply to: [231016690](https://github.com/dotnet/machinelearning/pull/1538#discussion_r231016690) [](ancestors = 231016690)